content/files/Akeneo PIM API.postman_collection.json

```diff
@@ -3133,6 +3333,86 @@
                 "rest",
                 "v1",
                 "product-models"
```

Comment:
We should also add the following query parameters on GET /api/rest/v1/product-models/{code}:
- with_quality_scores